benchmark/run_all_benchmarks.sh

```diff
@@ -173,9 +215,42 @@ generate_suite_sparse_input() {
 EOT
 }
 
+parse_matrix_list() {
+    local source_list_file=$1
+    local benchmark_list=""
+    local id=0
+    for mtx in $(cat ${source_list_file}); do
+        if [[ ! "$mtx" =~ ^[0-9]+$ ]]; then
+            if [[ "$mtx" =~ ^[a-zA-Z0-9_-]+$ ]]; then
+                id=$(${SSGET} -s "[ @name == $mtx ]")
+            elif [[ "$mtx" =~ ^[a-zA-Z0-9_-]+/[a-zA-Z0-9_-]+$ ]]; then
+                local group=$(echo $mtx | cut -d"/" -f1)
+                local name=$(echo $mtx | cut -d"/" -f2)
```

Comment:
```suggestion
            elif [[ "$mtx" =~ ^([a-zA-Z0-9_-]+)\/([a-zA-Z0-9_-]+)$ ]]; then
                local group="${BASH_REMATCH[1]}"
                local name="${BASH_REMATCH[2]}"
```
I prefer bash built-in function rather than system call although it does not hurt the performance much.